server/lib/s3backup/__init__.py

```diff
@@ -189,6 +183,35 @@ def head_bucket(self, Bucket=None):
     def list_objects(self, **kwargs):
         return self.connector.list_objects(**kwargs)
 
+    # response decoding - tbh is the header returned
+    # by get_tarball_header().
```

Comment:
In some places, line at line 199 below, the comment for the method follows the function declaration line.  In this case it is before.  Seems like it should follow the function definition.  What do you think?

Reply:
OK.